mscore/file.cpp

```diff
@@ -2540,76 +2540,80 @@ bool MuseScore::saveSvg(Score* score, const QString& saveName)
             p.translate(-r.topLeft());
 
       foreach (Page* page, score->pages()) {
```

Comment:
while touching the code, you could as well change the all the `foreach (a, b)` into `for (a : b)`